srcpkgs/mpg123-extras/template

```diff
@@ -0,0 +1,69 @@
+# Template file for 'mpg123-extras'
+# Sync with mpg123
+pkgname=mpg123-extras
+version=1.31.1
+revision=3
+build_style=gnu-configure
+# --with-cpu not necessary, upstream detects features correctly
+configure_args="--with-optimization=0 --with-default-audio=alsa"
+hostmakedepends="pkg-config"
+makedepends="alsa-lib-devel SDL2-devel jack-devel pulseaudio-devel sndio-devel"
+depends="mpg123-sdl mpg123-jack mpg123-pulseaudio mpg123-sndio"
+short_desc="Fast console MPEG audio decoder/player - extra plugins"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="LGPL-2.1-only"
+homepage="https://www.mpg123.org/"
+changelog="https://www.mpg123.org/cgi-bin/scm/mpg123/trunk/NEWS"
+distfiles="${SOURCEFORGE_SITE}/mpg123/mpg123-${version}.tar.bz2
+ https://www.mpg123.org/download/mpg123-${version}.tar.bz2"
+checksum="5dcb0936efd44cb583498b6585845206f002a7b19d5066a2683be361954d955a
+ 5dcb0936efd44cb583498b6585845206f002a7b19d5066a2683be361954d955a"
+
+if [ "$XBPS_TARGET_LIBC" = musl ]; then
+	configure_args+=" --disable-lfs-alias"
+fi
+
+post_install() {
+	# Prune everything except the plugins that will be installed
+	find ${DESTDIR} \
+		'(' -name output_sdl.so \
+			-o -name output_jack.so \
+			-o -name output_pulse.so \
+			-o -name output_sndio.so \
+		')' -o '(' -not -type d -delete ')'
+
+	# Main package cannot be empty
+	vdoc README
+}
+
+mpg123-sdl_package() {
+	depends="libmpg123>=${version}_${revision}"
```

Comment:
Please don't `>=_${revision}` because the `-extra` needs to be rebuilt whenever `mpg123` is rebuilt for no good reason. We probably want to limit to same version, too. 
```suggestion
	depends="libmpg123>=${version}_1<=${version}_999"
```

Reply:
Setting `<=` a high revision number is a bit hacky, but probably less horrible than the alternatives, which would be either hacking out a `next_version` variable so we could do `libmpg123>=${version}<${next_version}` (which would be fragile) or just dropping the `<=` (which might break some people's systems on upgrade). Done.